_schema.graphql

```diff
@@ -1407,8 +1407,12 @@ type ArtworkContextAuction implements Node {
     # A tz database time zone, otherwise falls back to `X-TIMEZONE` header
     timezone: String
   ): String
+
+  # A formatted description of when the auction starts or ends or if it has ended
+  formattedStartDateTime: String
   href: String
   name: String
+  initials(length: Int = 3): String
```

Comment:
I didn't expect these changes..

Reply:
This is a matter of rebasing. It was already updated in an earlier PR.